lambda/tests/test_github_usage.py

```diff
@@ -2,8 +2,8 @@
 import os
 
 import pytest
-import requests_mock
 
+import requests_mock
```

Comment:
isort has let you down here, `requsts_mock` should be in the second group of installed packages, not the third group of user modules

Reply:
It has, however as `isort` is run as part of the `Makefile test` command I'm going to let it do what it wants or we'll have to revert it everytime the tests are run.

Reply:
totally agreed. 
The fix would not be to hand-craft this line, but fix the makefile. But I also don't care too much.